backend/node/genvm/base.py

```diff
@@ -47,6 +53,15 @@ def safe_globals(override_globals: dict[str] = None):
         globals().update(old_globals)
 
 
+_FAKE_DECODED_DATA = object()
```

Comment:
what's this for? 

seems like you may want to check if some variables have or not been initialized, I think using `None` is a better approach

Reply:
I think no, because `None` can be encoded in calldata (even though it is invalid here). So I wish to distinguish between parsed calldata and unparsed

Reply:
and can `object()` be encoded in calldata?

Reply:
This object is checked with `is`, so it is checked for a reference, and other code can't get the same reference. This trick is common in java
Answering your question, python object can't, js object can I think (map from a string to something)